build.sbt

```diff
@@ -175,6 +175,7 @@ def configCommonTestSettings(scopes: String) = Seq(
   testOptions += Tests.Argument(TestFrameworks.JUnit, "-v"),
   libraryDependencies ++= Seq (
     "org.assertj" % "assertj-core" % "3.0.0" % scopes,
+    "com.commercetools.sdk.jvm.core" % "commercetools-test-lib" % "1.0.0-RC2" % scopes,
```

Comment:
Hmm, I'd rather suggest we create a test lib that can be used everywhere, this one is still tailored for the JVM SDK.